src/scripts/main.js

```diff
@@ -1,7 +1,96 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
+const game = new Game();
+const container = document.querySelector('.container');
+const gameField = document.querySelector('.game-field');
+const button = document.querySelector('.button');
+const gameScore = document.querySelector('.game-score');
+const messages = {
+  start: container.querySelector('.message-start'),
+  restart: container.querySelector('.message-restart'),
+  lose: container.querySelector('.message-lose'),
+  win: container.querySelector('.message-win'),
+};
 
-// Write your code here
+const fieldRows = [...gameField.querySelectorAll('.field-row')];
+const fieldCells = fieldRows.map((row) => [...row.children]);
+
+function displayGameField(state, cellElements) {
+  state.forEach((row, i) => {
+    row.forEach((cell, j) => {
+      const currentCell = cellElements[i][j];
+
+      currentCell.className = !cell
+        ? 'field-cell'
+        : `field-cell field-cell--${cell}`;
+      currentCell.innerHTML = !cell ? '' : cell;
+    });
+  });
+}
+
+function updateScore(score, elem) {
+  elem.innerHTML = score;
+}
+
+function showMessage() {
+  const gameStatus = game.getStatus();
+
+  for (const key in messages) {
+    if (Object.hasOwnProperty.call(messages, key)) {
+      const message = messages[key];
+
+      if (message) {
+        message.classList.toggle('hidden', key !== gameStatus);
+      }
+    }
+  }
+}
```

Comment:
The `showMessage` function uses a for-in loop to iterate over properties of an object, which is not recommended for iterating over arrays or array-like objects. Since `messages` is a plain object, it's fine here, but ensure that this is the intended behavior. Also, consider using `Object.keys` or `Object.entries` for a more functional approach.